src/main/java/uk/nhs/hee/tis/trainee/sync/config/MongoConfiguration.java

```diff
@@ -56,16 +56,14 @@ public void initIndexes() {
     placementIndexOps.ensureIndex(new Index().on("data.siteId", Direction.ASC));
 
     IndexOperations programmeMembershipIndexOps = template.indexOps(ProgrammeMembership.class);
-    programmeMembershipIndexOps.ensureIndex(new Index().on("data.programmeId", Direction.ASC));
-    programmeMembershipIndexOps.ensureIndex(new Index().on("data.curriculumId", Direction.ASC));
-    programmeMembershipIndexOps.ensureIndex(new Index().on("data.personId", Direction.ASC));
+    programmeMembershipIndexOps.ensureIndex(new Index().on("programmeId", Direction.ASC));
+    programmeMembershipIndexOps.ensureIndex(new Index().on("personId", Direction.ASC));
     Document keys = new Document();
-    keys.put("data.programmeId", 1);
-    keys.put("data.curriculumId", 1);
-    keys.put("data.personId", 1);
-    keys.put("data.programmeMembershipType", 1);
-    keys.put("data.programmeStartDate", 1);
-    keys.put("data.programmeEndDate", 1);
+    keys.put("programmeId", 1);
+    keys.put("personId", 1);
+    keys.put("programmeMembershipType", 1);
+    keys.put("programmeStartDate", 1);
+    keys.put("programmeEndDate", 1);
```

Comment:
This triggered a mongo error when I ran it locally, since an index with that name already exists (though it comprises a different set of fields). I guess we could simply manually delete the existing index from stage/prod before deploying - otherwise we'd need to delete the existing one, or create this new one with a different name.

Reply:
I was planning to manually drop the whole PM collection before deploying, since none of the existing data is migratable